ui/apps/platform/src/Containers/SystemHealth/UsageStatistics/UsageStatisticsForm.tsx

```diff
@@ -0,0 +1,168 @@
+                }}
+            >
+                <DescriptionListGroup>
+                    <DescriptionListTerm>Start date</DescriptionListTerm>
+                    <DescriptionListDescription>
+                        <DatePicker
+                            value={yyyyMMddFormat(startDate)}
+                            onChange={(_str, _, date) => {
+                                if (date) {
+                                    setStartDate(date);
+                                }
+                            }}
+                        />
+                    </DescriptionListDescription>
+                </DescriptionListGroup>
+                <DescriptionListGroup>
+                    <DescriptionListTerm>End date</DescriptionListTerm>
+                    <DescriptionListDescription>
+                        <DatePicker
+                            value={yyyyMMddFormat(endDate)}
+                            onChange={(_str, _, date) => {
+                                if (date) {
+                                    // Add 23 hours 59 minutes to include end date completely in the request.
+                                    date.setTime(date.getTime() + (24 * 60 - 1) * 60 * 1000);
+                                    setEndDate(date);
+                                }
+                            }}
+                        />
+                    </DescriptionListDescription>
+                </DescriptionListGroup>
+                <DescriptionListGroup>
+                    <DescriptionListTerm>CPU units</DescriptionListTerm>
+                    <DescriptionListDescription>{maxUsage.maxCpuUnits}</DescriptionListDescription>
+                </DescriptionListGroup>
+                <DescriptionListGroup>
+                    <DescriptionListTerm>Nodes count</DescriptionListTerm>
+                    <DescriptionListDescription>{maxUsage.maxNodes}</DescriptionListDescription>
+                </DescriptionListGroup>
+                <DescriptionListGroup>
+                    <DescriptionListTerm>CPU units collection date</DescriptionListTerm>
+                    <DescriptionListDescription>
+                        {maxUsage && maxUsage?.maxCpuUnitsAt
+                            ? yyyyMMddFormat(new Date(maxUsage.maxCpuUnitsAt))
+                            : '-'}
+                    </DescriptionListDescription>
+                </DescriptionListGroup>
+                <DescriptionListGroup>
+                    <DescriptionListTerm>Nodes count collection date</DescriptionListTerm>
+                    <DescriptionListDescription>
+                        {maxUsage && maxUsage?.maxNodesAt
+                            ? yyyyMMddFormat(new Date(maxUsage.maxNodesAt))
+                            : '-'}
+                    </DescriptionListDescription>
+                </DescriptionListGroup>
+            </DescriptionList>
+            <Grid hasGutter style={{ padding: '20px 0px 0px 0px' }}>
```

Comment:
We should be adding arbitrary spacing; instead, we should use the classes that PatternFly provides to keep a consistent look and feel throughout the app.

See https://v4-archive.patternfly.org/v4/utilities/spacing